utilities/transactions/write_prepared_txn_db.h

```diff
@@ -739,6 +768,22 @@ class WritePreparedCommitEntryPreReleaseCallback : public PreReleaseCallback {
         db_->AddCommitted(commit_seq + i, last_commit_seq);
       }
     }
+    bool delayed_exist =
```

Comment:
actually the whole MarkDelayedPreparedCommitted is unnecessary now since the latest changes will check every single eviction against delayed_prepared_.

Reply:
In that case, the commit entry will be added to the commit cache. At the read time, as long as the reader can find the commit entry in the commit cache they do not look to delayed_prepared_ so no more is require to ensure correctness. If the reader does not find the commit entry in the commit cache it means that it is evicted in the meanwhile, and this is also correct since we check all the evicted entires against the delayed_prepared_.

Reply:
Sorry, I was wanting to ask about the other case: what if the commit is not adding to commit cache, but old commit map directly? In that case it won't go through being evicted from commit cache. Who will add it to delayed_prepared_commits?